examples/singleton/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "scripts": {
+    "asbuild:untouched": "asc assembly/index.ts -b build/untouched.wasm -t build/untouched.wat --use abort=abort_proc_exit --sourceMap --debug",
+    "asbuild:optimized": "asc assembly/index.ts -b build/optimized.wasm -t build/optimized.wat --use abort=abort_proc_exit --sourceMap --optimize",
+    "asbuild": "npm run asbuild:untouched && npm run asbuild:optimized",
+    "test": "node tests"
+  },
+  "dependencies": {
+    "assemblyscript": "^0.14.8",
+    "@solo-io/proxy-runtime": "file:../.."
+  },
+  "devDependencies": {
+    "assemblyscript": "^0.14.8"
```

Comment:
Do we need this as a devDependency, when it's already in dependencies? (or vice-versa?)

Reply:
i think we only need devDependency; but i'm not sure.. that's why i just updated the version numbers so they all match